src/views/CoCurricularTranscript/Components/Experience/index.tsx

```diff
@@ -35,4 +33,19 @@ const formatDuration = ({ Job_Start_Date, Job_End_Date }: StudentEmployment) =>
   }
 };
 
+let prev_job_title = '';
+
+const newJobTitle = ({ Job_Department_Name, Job_Title }: StudentEmployment) => {
+  if (Job_Title != prev_job_title) {
+    prev_job_title = Job_Title;
+    if (Job_Department_Name == Job_Title.split(':')[0]) {
+      return Job_Title;
+    } else {
+      return Job_Department_Name + ', ' + Job_Title;
+    }
+  } else {
+    return '';
+  }
+};
```

Comment:
```suggestion
let prev_job_title = '';

const newJobTitle = ({ Job_Department_Name, Job_Title }: StudentEmployment) => {
  if (Job_Title != prev_job_title) {
    prev_job_title = Job_Title;
    return Job_Department_Name === Job_Title.split(':')[0] 
      ? Job_Title
      : `${Job_Department_Name}, ${Job_Title}`;
  }
  return '';
};
```

Couple things:
1) I'm pretty sure your variable `prev_job_title` as scoped variable can be finicky. @EjPlatzer probably has an opinion on this
2) there is some logic cleanup you can employ